src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerSearchService.kt

```diff
@@ -28,6 +32,7 @@ class PrisonerSearchService(
     const val RESULT_HITS_MAX = 1000
   }
 
+  @PreAuthorize("hasRole('GLOBAL_SEARCH')")
```

Comment:
could this be moved up to the class declaration instead since it is on all the public methods?